libs/components/popovers/src/lib/modules/shared/sky-popovers-resources.module.ts

```diff
@@ -23,6 +23,9 @@ const RESOURCES: { [locale: string]: SkyLibResources } = {
     skyux_dropdown_context_menu_with_content_descriptor_default_label: {
       message: 'Context menu for {0}',
     },
+    skyux_dropdown_context_menu_with_content_descriptor_el_default_label: {
+      message: 'Context menu for ',
```

Comment:
Did we decide to remove "for" from this description?

Reply:
No we decided to leave it for now. At least that was what I announced at the end of flex and no one disagreed with me